rust-server/src/conn.rs

```diff
@@ -38,16 +50,18 @@ impl dyn Connection {
                 let user_version: u64 =
                     tx.pragma_query_value(None, "user_version", |row| row.get(0))?;
                 if user_version == 0 {
-                    tx.execute_batch(include_str!("../sql/sqlite.sql"))?;
```

Comment:
This actually baked the schema into the exe. I don't know of a situation where a user would want to have a custom schema at runtime for these storage types.